examples/async-examples.ts

```diff
@@ -0,0 +1,53 @@
+import {RevAiApiClient, RevAiJobOptions} from 'revai-node-sdk';
+
+/* Initialize your client with your revai access token */
+const token = <YOUR-ACCESS-TOKEN>;
+var client = new RevAiApiClient(token);
+
+/* Get account details */
+var account = await client.getAccount();
+console.log(`Account: ${account.email}`);
+console.log(`Balance: ${account.balance_seconds} seconds`);
+
+/* Submit job and receive job object
+ * Media may be submitted from a local file or from a url by using:
```

Comment:
I find the wording in this comment confusing.

Also the placement of this comment is far from the function: `submitJobLocalFile`